rtscomponentkit/src/main/java/io/dolby/rtscomponentkit/data/RTSViewerDataStore.kt

```diff
@@ -73,18 +75,14 @@ class RTSViewerDataStore(
             startSubscribe()
         }
 
-        override fun onDisconnected() {
-            TODO("Not yet implemented")
-        }
-
-        override fun onActive(p0: String, p1: Array<out String>, p2: Optional<String>) {
+        override fun onActive(p0: String?, p1: Array<out String>?, p2: Optional<String>?) {
```

Comment:
p1, the list of tracks is always non null
p2, the optional for the sourceId can't be null (it's internal value) can